src/reactviews/pages/QueryResult/commandBar.tsx

```diff
@@ -23,8 +24,11 @@ const useStyles = makeStyles({
 });
 
 const CommandBar = () => {
-    const state = useContext(QueryResultContext);
-    const queryResultState = state?.state;
+    const context = useContext(QueryResultContext);
+    const queryResultState = context?.state;
+    if (context === undefined) {
```

Comment:
oh, sorry; my previous comment was a bit confusing:

if `context` always has `state`, then this check should go before we allocate memory for `queryResultState`.  If `state` might be undefined, then this check should be `state === undefined` instead.

Reply:
Oh ok makes sense, I think I was also trying to address all the comments quickly and got a bit confused along with the renaming.